internal/provider/job_resource_test.go

```diff
@@ -33,7 +33,7 @@ const (
 func TestJobResourceSchema(t *testing.T) {
 	t.Parallel()
 
-	ctx := context.Background()
+	ctx := t.Context()
```

Comment:
Thank you for tidying these up to use `t.Context()` instead of `context.Background()` ❤️